src/wallet/rpcwallet.cpp

```diff
@@ -3525,9 +3525,17 @@ UniValue z_sendmany(const UniValue& params, bool fHelp)
     o.push_back(Pair("fee", std::stod(FormatMoney(nFee))));
     UniValue contextInfo = o;
 
+    // Contextual transaction we will build on
+    CMutableTransaction contextualTx = CreateNewContextualCMutableTransaction(
+        Params().GetConsensus(),
+        1 + chainActive.Height());
```

Comment:
`chainActive.Height() + 1`

Reply:
Looks like this is still unchanged in this particular spot.

Reply:
Specsavers for me.